cg/meta/upload/gisaid/models.py

```diff
@@ -45,43 +43,15 @@ class GisaidSample(BaseModel):
     covv_subm_lab_addr: Optional[str] = "171 76 Stockholm, Sweden"
     covv_authors: Optional[str] = " ,".join(AUTHORS)
 
-    @validator("lab", pre=True)
-    def lab_info(cls, v):
-        if v == "Stockholm":
-            return Lab(
-                city="Stockholm",
-                address="171 76 Stockholm, Sweden",
-                region_nr="01",
-                institute="Karolinska University Hospital",
-            )
-        elif v == "Visby":
-            return Lab(
-                city="Visby",
-                address="621 84 Visby, Sweden",
-                region_nr="09",
-                institute="LaboratorieMedicinskt Centrum Gotland",
-            )
-        raise ValueError("must be Stockholm or Visby")
-
     @validator("covv_location", always=True)
     def parse_location(cls, v, values):
-        lab = values.get("lab")
-        return f"Europe/Sweden/{lab.city}"
-
-    @validator("covv_orig_lab_addr", always=True)
-    def parse_orig_lab_addr(cls, v, values):
-        lab = values.get("lab")
-        return lab.address
-
-    @validator("covv_orig_lab", always=True)
-    def parse_orig_lab(cls, v, values):
-        lab = values.get("lab")
-        return lab.institute
+        region: str = values.get("region")
+        return f"Europe/Sweden/{region}"
 
     @validator("covv_subm_sample_id", always=True)
     def parse_subm_sample_id(cls, v, values):
-        lab = values.get("lab")
-        return f"{lab.region_nr}_SE100_{v}"
+        region_code = values.get("region_code")
+        return f"{region_code}_SE100_{v}"
```

Comment:
Is the SE100 not lab code? in that case it is not always SE100 and can also be fetched from lims

Reply:
According to Robert it should allways be SE100. I can double check. with him

Reply:
Robert comfirmed: Allways SE100